rust_src/remacs-sys/lib.rs

```diff
@@ -736,6 +736,8 @@ pub struct Lisp_Char_Table {
 
 #[repr(C)]
 pub struct Lisp_Process {
+    pub header: Lisp_Vectorlike_Header,
```

Comment:
So... nobody was using the members before?

Reply:
Apparently. I was getting `tty_name` instead of `name` there.